src/App.test.jsx

```diff
@@ -1,4 +1,23 @@
+import { render, screen, waitFor } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+import userEvent from '@testing-library/user-event';
+import App from './App';
 
-test('Should render the header', () => {
+describe('App', () => {
+  it('should render the component Header', async () => {
+    render(
+      <MemoryRouter>
```

Comment:
since we aren't rendering a link anywhere we don't actually need to use MemoryRouter here.